src/main/java/net/dv8tion/jda/api/requests/GatewayIntent.java

```diff
@@ -69,6 +144,17 @@ public int getOffset()
         return set;
     }
 
+    /**
+     * Converts the given intents to a
```

Comment:
This sentence is incomplete. I'm not sure if this is intentional, but it doesn't read very nicely.